### DISCUSSION TITLE
Automatic purchase push notification reminder

### DISCUSSION THREAD

**Discussion by jsaristi** [inspireui/support#5666] [New Feature/Ideas Requests]
Automatic purchase push notification reminder

This functionality is very important and would help to double the sales within the application and it is to place that automatically every 30 or 45 days depending on the user's personalized option, the application sends a push notification with the image, the name of the user and the product reminding you to repurchase the product you purchased in the past. The notification might say something like this:

Hello "Simon", buy again the "Product Name" that you liked so much.

This functionality would be great for everyone as it would help generate a lot more sales and brand engagement.

It would also be great to send push notifications to abandoned carts.

They should push more notifications that are so important today.

![5SuccessfulPushBlog3](https://user-images.githubusercontent.com/29686082/201502418-f4858418-1b9d-4332-a9e3-5ec6a2f8983e.png)

**Comment by jsaristi** [comment]
Please all vote this genial feature! Thanks!

**Comment by ameenbadri7** [comment]
up +1

**Comment by osman-kocak** [comment]
Up

**Comment by futservices** [comment]
+1

**Comment by jooooo-dot** [comment]
Up !!

**Comment by osman-kocak** [comment]
@minhcasi  @HanielQC
